page-optimize.php

```diff
@@ -132,9 +132,18 @@ function page_optimize_sanitize_exclude_field( $value ) {
 	return implode( ',', $sanitized_values );
 }
 
-require_once __DIR__ . '/settings.php';
-require_once __DIR__ . '/concat-css.php';
-require_once __DIR__ . '/concat-js.php';
+function page_optimize_init() {
+	// Bail if we're in customizer
+	global $wp_customize;
+	if ( isset( $wp_customize ) ) {
+		return;
+	}
+
+	require_once __DIR__ . '/settings.php';
+	require_once __DIR__ . '/concat-css.php';
+	require_once __DIR__ . '/concat-js.php';
 
-// Disable Jetpack photon-cdn for static JS/CSS
-add_filter( 'jetpack_force_disable_site_accelerator', '__return_true' );
+	// Disable Jetpack photon-cdn for static JS/CSS
+	add_filter( 'jetpack_force_disable_site_accelerator', '__return_true' );
+}
+add_action( 'plugins_loaded', 'page_optimize_init' );
```

Comment:
It's kind-of late loading our plugin, but haven't seen any performance differences between this and loading as we did before.

The thing with the old way is... I can't detect customizer (or haven't figured out a way to).